src/test/operators/maintenance/drop_view_test.cpp

```diff
@@ -44,15 +44,28 @@ TEST_F(DropViewTest, DeepCopy) {
   EXPECT_EQ(copy->get_output(), nullptr);
 }
 
-TEST_F(DropViewTest, CanDropViews) {
-  EXPECT_TRUE(StorageManager::get().has_view("view_name")) << "View not found";
-
-  auto dv = std::make_shared<DropView>("view_name");
+TEST_F(DropViewTest, Execute) {
+  auto dv = std::make_shared<DropView>("view_name", false);
   dv->execute();
 
   EXPECT_EQ(dv->get_output()->row_count(), 0u) << "DropView returned non-empty table";
 
   EXPECT_FALSE(StorageManager::get().has_view("view_name")) << "View was not removed";
 }
 
+TEST_F(DropViewTest, ExecuteWithIfExists) {
+  auto dv_1 = std::make_shared<DropView>("view_name", true);
+  dv_1->execute();
+
+  EXPECT_EQ(dv_1->get_output()->row_count(), 0u) << "DropView returned non-empty table";
```

Comment:
Admirable, but we usually do not supply additional messages in tests - `dv_1->get_output()->row_count() != 0` already tells us what the message says. Remember - comments and messages are essentially untested code and therefore error prone and better avoided if possible ;)

Reply:
I just wanted to keep consistency within in this test, but you're right of course. I'll just delete these additional messages